extensions/git/src/commands.ts

```diff
@@ -1236,17 +1236,29 @@ export class CommandCenter {
 		await repository.ignore(uris);
 	}
 
+	@command('git.stashIncludeUntracked', { repository: true })
+	async stashIncludeUntracked(repository: Repository): Promise<void> {
+		if (repository.workingTreeGroup.resourceStates.length === 0) {
+			window.showInformationMessage(localize('no changes stash', "There are no changes to stash."));
```

Comment:
Can you simplify this like that:
`return window.showInformationMessage(localize('no changes stash', "There are no changes to stash."));`